docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -1,129 +1,161 @@
 # CPU device {#openvino_docs_OV_UG_supported_plugins_CPU}
 
 
-## Introducing the CPU Plugin
-The CPU plugin was developed to achieve high performance of neural networks on CPU, using the Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN).
+The plugin allows to achieve high performance of inference using CPU, using the Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN). It utilizes Intel® Threading Building Blocks (Intel® TBB) to parallelize calculations. For performance considerations, please refer to the [Optimization Guide](../../optimization_guide/dldt_optimization_guide.md).
 
-Currently, the CPU plugin uses Intel® Threading Building Blocks (Intel® TBB) in order to parallelize calculations. Please refer to the [Optimization Guide](../../optimization_guide/dldt_optimization_guide.md) for associated performance considerations.
+Its set of supported layers can be expanded with [the Extensibility mechanism](../Extensibility_DG/Intro.md).
 
-The set of supported layers can be expanded with [the Extensibility mechanism](../Extensibility_DG/Intro.md).
+The CPU plugin supports inference on:
+- Intel® Xeon® with Intel® Advanced Vector Extensions 2 (Intel® AVX2),
+- Intel® Advanced Vector Extensions 512 (Intel® AVX-512) and AVX512_BF16,
+- Intel® Core™ Processors with Intel® AVX2, 
+- Intel Atom® Processors with Intel® Streaming SIMD Extensions (Intel® SSE).
 
-## Supported Platforms
-
-OpenVINO™ toolkit, including the CPU plugin, is officially supported and validated on the following platforms:
-
-| Host              | OS (64-bit)                              |
-| :---              | :---                                     |
-| Development       | Ubuntu* 18.04 or 20.04, CentOS* 7.6, MS Windows* 10, macOS* 10.15 |
-| Target            | Ubuntu* 18.04 or 20.04, CentOS* 7.6, MS Windows* 10, macOS* 10.15 |
+To see which configuration is used by a layer, you can use the `-pc` flag for samples. 
+It shows execution statistics that you can use to get information about layer name, layer type, 
+execution status, execution time, and the type of the execution primitive.
 
-The CPU plugin supports inference on Intel® Xeon® with Intel® Advanced Vector Extensions 2 (Intel® AVX2), Intel® Advanced Vector Extensions 512 (Intel® AVX-512), and AVX512_BF16, Intel® Core™
-Processors with Intel® AVX2, Intel Atom® Processors with Intel® Streaming SIMD Extensions (Intel® SSE).
+**First inference latency**  
+To reduce first inference latency, the plugin utilizes the ngraph function serialization mechanism to enable model caching. Caching a compiled blob eliminates the impact of Common Transformations and Low Precision Transformations on first inference latency of the following runs.
```

Comment:
`ngraph` name is removed from 2022.1. 
Maybe `the model serialization mechanism` ?

Reply:
That's a nice rewording. Change included.